modules/ROOT/pages/manual-setup.adoc

```diff
@@ -84,6 +84,9 @@ The following instructions use URLs for the US cloud that begin with `+https://a
 . Set *Token Endpoint Url* to `+https://anypoint.mulesoft.com/accounts/api/v2/oauth2/token+`.
 . Set *Issuer* to the client ID of the connected app you saved previously.
 . Set *Per User Subject* to the following formula.
```

Comment:
Where is "the following formula"?

Reply:
These are the formulas on lines 94 and 99.